src/fips.c

```diff
@@ -95,12 +96,49 @@ static int rsa_kat(void);
 static int ecdsa_kat(void);
 static int ecdh_kat(void);
 
+static int function_supported_via_cpacf(unsigned int func)
+{
+	ica_adapter_handle_t ah;
+	libica_func_list_element* libica_func_list = NULL;
+	unsigned int count;
+	size_t i;
+	int ret = 0;
+
+	if (ica_open_adapter(&ah))
+		return ret;
+
+	if (ica_get_functionlist(NULL, &count) != 0)
+		return ret;
+
+	libica_func_list = malloc(sizeof(libica_func_list_element) *count);
```

Comment:
Please add a space between * and count.
`malloc(sizeof(libica_func_list_element) * count);`

Reply:
OK, done.